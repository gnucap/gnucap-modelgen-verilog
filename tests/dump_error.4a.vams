`include "sub/disciplines.few"
`pragma modelgen nodump-module
`pragma modelgen noexpand-paramset

module test(n, p)
	electrical n, p;
	analog begin
		XS(n, p) <+ 17.
	end
endmodule
