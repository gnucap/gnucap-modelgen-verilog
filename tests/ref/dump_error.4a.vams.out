... ical n, p;
 analog begin
  XS(n, p) <+ 17.
             ^ ? bad access
